Reading List/Controllers/BookController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  BookController.swift
+//  Reading List
+//
+//  Created by Bryan Cress on 5/19/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+
+class BookController {
+    // Read
+    var books: [Book] = []
+    var readbooks: [Book] {
+        books.filter{
+            $0.hasBeenRead == true }
+    }
+    var unreadbooks: [Book] {
+        books.filter{
+            $0.hasBeenRead == false }
+    }
+    
+    // Create
+    func createBook(title: String, reasonToRead: String, hasBeenRead: Bool) {
+        let book = Book(title: title, reasonToRead: reasonToRead, hasBeenRead: hasBeenRead)
+
+        books.append(book)
+        savetoPersistentStore()
+    }
+    
+    //Upload
+    
+    
+    //Delete
+    func deleteBook(book: Book) {
+        // find book passed in as a parameter in book aray
+        guard let index = books.firstIndex(of: book) else { return }
+        
+        //get books array/remove specific book from books aray
+        books.remove(at: index)
+    }
+    
+    
+    //Update books object hasBeenRead property
+    func updateHasBeenRead(for book: Book) {
+        if book.hasBeenRead == true {
+            books.append(book)
+            guard let index = books.firstIndex(of: book) else { return }
+            books.remove(at: index)
+            savetoPersistentStore()
+        }
+    }
+    
+    //Update edit the books title and/or reasonToRead properties
+    func editBook(for book: Book) {
+        guard let index = books.firstIndex(of: book) else { return }
+        books[index].title = "\(book.title)"
+        books[index].reasonToRead = "\(book.reasonToRead)"
+        savetoPersistentStore()
+    }
+    
+    
+    
```

Comment:
Let's trim down on this whitespace. It's good to use to keep code clean and readable, but too much and it causes a lot of unnecessary scrolling, and it will get worse as projects and code files get bigger.